Source/Plugins/Core/com.equella.core/js/__tests__/tsrc/search/SearchPage.test.tsx

```diff
@@ -308,13 +301,52 @@ describe("Hide Refine Search controls", () => {
     jest.clearAllMocks();
   });
 
-  const getOwnerSelector = () =>
-    queryRefineSearchComponent(page.container, "OwnerSelector");
-  const getDateSelector = () =>
-    queryRefineSearchComponent(page.container, "DateRangeSelector");
-  const getStatusSelector = () =>
-    queryRefineSearchComponent(page.container, "StatusSelector");
+  it("Should contain the correct controls", async () => {
+    const pageContainer = page.container;
+    const findCollapsibleSection = page.container.getElementsByClassName(
+      "collapsibleRefinePanel"
+    );
+    expect(findCollapsibleSection).toHaveLength(1);
+
+    const collapsibleSection = findCollapsibleSection.item(0);
+
+    expect(collapsibleSection).toContainElement(
+      queryOwnerSelector(pageContainer)
+    );
+    expect(collapsibleSection).toContainElement(
+      queryDateRangeSelector(pageContainer)
+    );
+    expect(collapsibleSection).toContainElement(
+      querySearchAttachmentsSelector(pageContainer)
+    );
+    expect(collapsibleSection).not.toContainElement(
+      queryCollectionSelector(pageContainer)
+    );
+  });
+
+  it("Should change button text when clicked", async () => {
+    const expansionButton = page.getByText(
+      languageStrings.common.action.showMore
+    );
+    fireEvent.click(expansionButton);
+    expect(expansionButton).toHaveTextContent(
+      languageStrings.common.action.showLess
+    );
+  });
+});
+describe("Hide Refine Search controls", () => {
```

Comment:
Good to have a line space here, as you're into a whole new logical block.
```suggestion

describe("Hide Refine Search controls", () => {
```